Assets/sass/_variables.scss

```diff
@@ -17,6 +18,7 @@ $font-xl: 1.5rem;
 $font-xxl: 1.7rem;
 $font-xxxl: 2rem;
 $font-title: 3rem;
+$font-titlelarge: 4rem;
```

Comment:
change to font-title-large for consistency, and alt-blue for the other line change in this file

Reply:
applied, and to alt-blue as well